app/_src/guides/excluding-traffic-from-transparent-proxy-redirection.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Excluding Traffic from Transparent Proxy Redirection
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+
+This guide will show you how to exclude specific types of traffic from being redirected by the transparent proxy. It includes instructions for both **Universal** and a **Kubernetes** modes and covers different types of traffic you might want to exclude, such as:
+
+- [Exclude traffic on certain ports (incoming & outgoing)](#exclude-traffic-on-certain-ports)
+- [Exclude traffic to and from specific IP addresses](#exclude-traffic-to-and-from-specific-ip-addresses)
+
+Use this guide to control which traffic the transparent proxy intercepts and which it allows through directly.
+
+## Terminology overview
+
+- **Workload**: In this guide, **workload** refers to an application running in a Kubernetes cluster, usually represented by a [**Pod**](https://kubernetes.io/docs/concepts/workloads/pods/). Kubernetes resources like [**Deployment**](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), [**ReplicaSet**](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/), and [**StatefulSet**](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) are used to define and manage these workloads, resulting in one or more **Pods** where the application actually runs.
+
+- **ConfigMap**: In this guide, **ConfigMap** refers to the Kubernetes [**ConfigMap**](https://kubernetes.io/docs/concepts/configuration/configmap/) resource. It’s used to store configuration data as key-value pairs that can be easily accessed by other resources in the cluster, such as Pods, Deployments, and StatefulSets.
+
+- **Service**: In this guide, **service** means the application running in a Universal mode environment where the transparent proxy will be installed. This term does not refer to Kubernetes [**Service**](https://kubernetes.io/docs/concepts/services-networking/service/) resources.
+
+## Prerequisites
+
+1. **Familiarity with transparent proxy:** You should already understand what a transparent proxy is, how to install it, and how to configure its settings. 
+
+2. **Set up {{ Kuma }}**: Ensure {{ Kuma }} is installed and running.
+
+   {% capture prerequisites-setup-kuma %}
+   {% tabs %}
+   {% tab Kubernetes %}
+   Follow the [Kubernetes Quickstart]({{ docs }}/quickstart/kubernetes-demo/) guide to set up a zone control plane and demo application.
+   {% endtab %}
+   {% tab Universal %}
+   Confirm that all necessary components are up and running. The following resources may be useful:
```

Comment:
wouldn't it be enought. to just link to universal quickstart or docker quickstart?